### ISSUE TITLE
False positive on useless parentheses for new ZipArchive

### ISSUE THREAD

**Issue by adrhumphreys** [slevomat/coding-standard#567]
When you want to use a class such as `ZipArchive` you'll usually initialise it such as:
```php
use ZipArchive;
$zip = new ZipArchive();
```
Then later in your code do things with the zip such as:
``` php
$zip->open($fileLocation);
$zip->extractTo($extractLocation);
$zip->close();
```
The problem being that when you initialise `ZipArchive` it doesn't have a constructor so there's nothing to pass through. Therefore the parentheses are "useless", but you also can create a new instance of a class without parentheses. So with both of those rules you can't win.

This could me being new and all 👍

**Comment by kukulich** [comment]
I'm not able to reproduce the error: https://github.com/slevomat/coding-standard/commit/bedc3a242bdb3738e24b513b12f75dcc6746715f
